.travis/post_before_script.sh

```diff
@@ -18,4 +18,16 @@ cmd_prefix bash -c "django-admin shell -c \"${CREATE_SIGNING_SERVICE}\""
 echo "machine pulp
 login admin
 password password
-" > ~/.netrc
\ No newline at end of file
+" > ~/.netrc
+
+if [[ "$TEST" == 'pulp' || "$TEST" == 'performance' || "$TEST" == 's3' ]]; then
+    # replace a default origin in the fixture by the origin set by Travis
+    FIXTURES_CONTAINER_ID=$(docker ps -a | grep 'quay.io/pulp/pulp-fixtures:latest' | awk "{print \$1}")
+    FIXTURE_ORIGIN=$(jq -r '.custom.fixtures_origin' ~/.config/pulp_smash/settings.json | sed -r 's/\//\\\//g')
+
+    REPLACE_ORIGIN="sed -i -E 's/https?:\/\/.+?:[0-9]+\/fixtures\//${FIXTURE_ORIGIN}/g' /usr/share/nginx/html/fixtures/rpm-mirrorlist-good"
+    docker exec -ti ${FIXTURES_CONTAINER_ID} sh -c "${REPLACE_ORIGIN}"
+
+    REPLACE_ORIGIN="sed -i -E 's/https?:\/\/.+?:[0-9]+\/fixtures\//${FIXTURE_ORIGIN}/g' /usr/share/nginx/html/fixtures/rpm-mirrorlist-bad"
+    docker exec -ti ${FIXTURES_CONTAINER_ID} sh -c "${REPLACE_ORIGIN}"
+fi
```

Comment:
I have a couple PRs that should set the base url in mirror lists:

https://github.com/pulp/pulp-fixtures/pull/164
https://github.com/pulp/plugin_template/pull/233

Reply:
This should enable it: https://github.com/pulp/pulp_rpm/pull/1750